superset/db_engine_specs.py

```diff
@@ -550,6 +575,11 @@ def adjust_database_uri(cls, uri, selected_schema=None):
             uri.database = database
         return uri
 
+    @classmethod
+    def get_datatype(cls, type_code):
```

Comment:
This method is identical to the base class and can be removed, no?

Reply:
Yes that's the case, I know that Presto should be this method, but I'm unclear on what the base class should do to capture the most common use cases.